issue_15_tailwindcss/main.js

```diff
@@ -17,7 +18,7 @@ function removeLoading() {
 }
 
 function removeShowClassOfmodal() {
-    modal.classList.remove('show');
+    modal.classList.add('opacity-0', 'invisible');
 }
```

Comment:
[ IMO ]
With the change to tailwind CSS, I thought the function names did not match the content.

Reply:
https://github.com/yoshikawa-akr/js-lessons/pull/16/commits/909740e0b049560484a5ebe62b067ee7dc9ce9c9
Thank you for pointing this out.
You are exactly right:sparkles:
This time, TailwindCSS has put together the styles as custom CSS, so I have changed the function names and their processing accordingly.